portfolio/src/main/webapp/index.html

```diff
@@ -160,6 +162,27 @@ <h4 class="experience_header">Past Positions</h4>
             </div>
         </div>
 
+        <!-- Comments Section -->
+        <div id="Comments" class="tabcontent">
+            <form action="/data" method="POST">
+
+                <p>Name: </p>
+                <input type="text" name="comment_name">
+                <!-- <p>Comment: <input type="text" class="long_text" name="comment_content"></p> -->
+                <p>Comment: </p>
+                <textarea type="text" name="comment_content" rows="2"></textarea>
+
+                <input type="submit" />
+            </form>
+            <hr>
+            <!-- <div id="comments_container"> -->
```

Comment:
Same comment about checking in commented-out code applies to HTML, though perhaps not as strictly. 🙂